codes/quantum/qudits/modular-qudit-513.yml

```diff
@@ -0,0 +1,48 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+
+# code id, physical, logical are all lower case
+code_id: modular-qudit-513
+physical: qudits
+logical: qudits
+
+name: 'Modular qudit [[5,1,3]] code'
+introduced: '\cite{arXiv:quant-ph/9702033v3}'
+
+description: 'Qubits are defined in the system with spin equal to \(1/2\). This QECC is constructed for particles with spin higher than \(1/2\) using group-theoretical methods. Each $N$ state quantum particle (a particle with local dimension \(N\)) is encoded as five \(N\) state quantum registers. This code is optimal because it is not possible to correct a general quantum error involving exactly one quantum register by encoding each quantum particle by four (or less) quantum registers.'
```

Comment:
This can be made much more concise by defining everything in terms of qudits instead of N state particles. Also you should say here that this generalizes the 5-qubit code.

Reply:
Right. Here is a qudit family that extends a qubit code; all qudit info is passed on from the parents, so there is no need to be too verbose. Also, interpretation in terms of spin is not necessary for this code. https://errorcorrectionzoo.org/c/qudit_hamming_css